pandas/io/rdata/__init__.py

```diff
@@ -0,0 +1 @@
+from pandas.io.rdata.rdata_reader import read_rdata  # noqa
```

Comment:
Can you remove the ```# noqa```, and have a ```__all__ = ["read_rdata"]```?

Reply:
Good suggestion. Will change. I followed `pandas.io.sas` here.